ichnaea/webapp/tests.py

```diff
@@ -136,24 +139,16 @@ def test_post(self, app):
         assert res.content_type == "application/json"
         assert res.json["status"] == "OK"
 
-    def test_options(self, app):
```

Comment:
The service only supports ``OPTIONS`` for endpoints that need to support [pre-flight CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#Preflighted_requests), so this test now fails.